platformio/commands/account/command.py

```diff
@@ -194,5 +190,85 @@ def account_update(current_password):
 @cli.command("show", short_help="PIO Account information")
 @click.option("--offline", is_flag=True)
 @click.option("--json-output", is_flag=True)
-def account_show(**kwargs):
-    pioplus_call(sys.argv[1:])
+def account_show(offline, json_output):
+    client = AccountClient()
+    try:
+        info = client.get_account_info(offline)
+        if json_output:
+            return click.echo(json.dumps(info))
+        click.echo()
+        if info.get("profile"):
+            print_profile(info["profile"])
+        if info.get("packages"):
+            print_packages(info["packages"])
+        if info.get("subscriptions"):
+            print_subscriptions(info["subscriptions"])
+        return click.echo()
+    except exception.AccountNotAuthenticated as e:
+        return click.secho(str(e), fg="yellow",)
+
+
+def print_profile(profile):
+    click.echo("Profile:")
+    if profile.get("username"):
+        click.echo("- Username: %s" % profile["username"])
+    if profile.get("email"):
+        click.echo("- Email: %s" % profile["email"])
+    if profile.get("first_name"):
+        click.echo("- First name: %s" % profile["first_name"])
+    if profile.get("last_name"):
+        click.echo("- Last name: %s" % profile["last_name"])
+
+
+def print_packages(packages):
+    click.echo()
+    click.echo("Package%s:" % ("s" if len(packages) > 1 else ""))
+    for package in packages:
+        click.echo("- %s" % package.get("name"))
+        expire = "-"
+        if "subscription" in package:
+            expire = datetime.datetime.strptime(
+                (
+                    package["subscription"].get("end_at")
+                    or package["subscription"].get("next_bill_at")
+                ),
+                "%Y-%m-%dT%H:%M:%SZ",
+            ).strftime("%Y-%m-%d")
+        click.echo("  Expire: %s" % expire)
+        permissions = []
+        for key in package:
+            if not key.startswith("service.") or package[key].get("title"):
```

Comment:
Why? This is a service but there is no title - it's Ok for us! Means, this is a string. Show it!